frontend/src/views/CreateAccountPageView.tsx

```diff
@@ -29,11 +30,27 @@ const CreateAccountPage: React.FC = () => {
 
 	const handleLogin = async (e: FormEvent<HTMLFormElement>) => {
 		e.preventDefault(); // prevent refresh
-
+		const isValidUsername = (username: string): boolean => /^[a-zA-Z0-9]+$/.test(username);
```

Comment:
1. avoid magic check
2. second jr's point: consider exporting constant or function for regex checking